ee-9/feature-pack/pom.xml

```diff
@@ -743,6 +748,14 @@
                     <groupId>${project.groupId}</groupId>
                     <artifactId>wildfly-mail</artifactId>
                 </exclusion>
+                <exclusion>
+                    <groupId>${project.groupId}</groupId>
+                    <artifactId>wildfly-messaging-injection</artifactId>
```

Comment:
This should be 'wildfly-messaging-activemq-injection'

The missing 'activemq' is why you still needed to exclude this.